src/features/pages/home/markets/bottom-text/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import Container from 'features/components/atoms/container'
+import Button from 'features/components/atoms/button'
+
+const BottomText = () => {
+    return (
+        <Container.Fluid align="center" pt="10x" md={{ pt: '19x' }}>
```

Comment:
```suggestion
        <Flex.Box
            direction="col"
            container="fluid"
            justify="center"
            align="center"
            pt="10x"
            md={{ pt: '19x' }}
        >
            <Typography.Paragraph size="large" mb="10x" textcolor="black">
                <Localize translate_text="_t_Want to know more about CFD trading conditions for the instruments we offer?_t_" />
            </Typography.Paragraph>
            <Button.Primary>
                <Localize translate_text="_t_Check trading specs_t_" />
            </Button.Primary>
        </Flex.Box>
```
`align` prop is invalid